docs/conf/portland/2019/news/thanks-recap.rst

```diff
@@ -0,0 +1,93 @@
+:template: {{year}}/generic.html
+:banner: _static/{{year}}/assets/headers/venue.png
+
+.. post:: May 30, 2019
+   :tags: recap, thanks, portland-2019
+
+Conference highlights and thanks to the community!
+--------------------------------------------------
+
+Hey everyone,
+
+The Write the Docs team is now emerging from the post-conference
+recovery period, and we wanted to send out a quick email to thank
```

Comment:
The blog post is not an e-mail. Maybe just "and we wanted to thank everyone for helping" ?

Reply:
Or just send a quick thank you